collector/infiniband_linux.go

```diff
@@ -62,6 +63,18 @@ func NewInfiniBandCollector() (Collector, error) {
 		"unicast_packets_transmitted_total":   {"unicast_xmit_packets", "Number of unicast packets transmitted (including errors)"},
 	}
 
+	// Deprecated counters for some older versions of InfiniBand drivers.
+	i.legacyCounters = map[string]infinibandMetric{
+		"legacy_multicast_packets_received_total":    {"port_multicast_rcv_packets", "Number of multicast packets received"},
+		"legacy_multicast_packets_transmitted_total": {"port_multicast_xmit_packets", "Number of multicast packets transmitted"},
+		"legacy_data_received_bytes":                 {"port_rcv_data_64", "Number of data octets received on all links, represented by a 64 bit number"},
```

Comment:
Does the "represented by a 64 bit number" part matter for users of this Prometheus exporter?

Also I think this needs a `_total` suffix.